packages/@lwc/features/src/__tests__/flags.spec.ts

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const pluginTester = require('babel-plugin-tester');
+const plugin = require('../babel-plugin');
+
+const nonProdTests = {
+    'should transform boolean-true feature flags': {
+        code: `
+            import { ENABLE_FEATURE_TRUE } from '@lwc/features';
+            if (ENABLE_FEATURE_TRUE) {
+                console.log('ENABLE_FEATURE_TRUE');
+            }
+        `,
+        output: `
+            import { ENABLE_FEATURE_TRUE, runtimeFlags } from '@lwc/features';
+
+            if (runtimeFlags.ENABLE_FEATURE_TRUE) {
+              console.log('ENABLE_FEATURE_TRUE');
+            }
+        `,
+    },
+    'should transform boolean-false feature flags': {
+        code: `
+            import { ENABLE_FEATURE_FALSE } from '@lwc/features';
+            if (ENABLE_FEATURE_FALSE) {
+                console.log('ENABLE_FEATURE_FALSE');
+            }
+        `,
+        output: `
+            import { ENABLE_FEATURE_FALSE, runtimeFlags } from '@lwc/features';
+
+            if (runtimeFlags.ENABLE_FEATURE_FALSE) {
+              console.log('ENABLE_FEATURE_FALSE');
+            }
+        `,
+    },
+    'should not tramsform feature flags unless the if-test is a plain indentifier': {
+        code: `
+            import { ENABLE_FEATURE_TRUE, ENABLE_FEATURE_FALSE, ENABLE_FEATURE_NULL } from '@lwc/features';
+            if (ENABLE_FEATURE_NULL === null) {
+                console.log('ENABLE_FEATURE_NULL === null');
+            }
+            if (isTrue(ENABLE_FEATURE_TRUE)) {
+                console.log('isTrue(ENABLE_FEATURE_TRUE)');
+            }
+            if (!ENABLE_FEATURE_FALSE) {
+                console.log('!ENABLE_FEATURE_FALSE');
+            }
+        `,
+        output: `
+            import { ENABLE_FEATURE_TRUE, ENABLE_FEATURE_FALSE, ENABLE_FEATURE_NULL, runtimeFlags } from '@lwc/features';
+
+            if (ENABLE_FEATURE_NULL === null) {
```

Comment:
I'm not sure why this is not transformed to `runtimeFlags.ENABLE_FEATURE_NULL`

Reply:
The transform is dead simple and only kicks in if the if-test is a simple identifier. Since the if-test is a binary expression, we ignore it.